GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/SpeciesListActivityFragment.java

```diff
@@ -66,10 +67,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
              * is called upon a click in the speciesList
              */
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
-                //todo make this bring up a dialog box
-                //for now it's just going to be a toast to see if it was clicked
-                Toast.makeText(getActivity(),  adapter.getItem(position), Toast.LENGTH_SHORT).show();
+
+                //Toast.makeText(getActivity(),  adapter.getItem(position), Toast.LENGTH_SHORT).show();
```

Comment:
The comments explaining this call were removed. Now the call itself commented out with no description of why it still exists. It seemed to be a method for testing. Will this be used in the future? A little clarity would help.